### PR TITLE
:bug: Source Posthog: Fix events.event type

### PR DESCRIPTION
**Pull request by dynalz**
## What

Posthog Event schema has wrong type on event column, the type should be a string as can be seen at official posthog api:
https://posthog.com/docs/api/events - Both get (for filtering) and response values from event key are string.

Checking Posthog frontend they are also all labeled as string:
image

The issue this is causing is making the source channel write the event column in json instead of string. Making it difficult to query since it's not really a json either
Example of Posthog to Postgres behavior on image below:
image

## How
Change event typing from `["string", "object"]` to `string`

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Comment by marcosmarxm** [comment]
@dynalz can you sign the CLA?

**Comment by dynalz** [comment]
> @dynalz can you sign the CLA?

done

**Review by marcosmarxm — APPROVED** [review]
Thanks @dynalz

**Comment by marcosmarxm** [comment]
Hello @dynalz thanks for the contribution and patience to get it merged. This week the codebase is freeze because of holidays, but I'll proceeded next week to get it merged.

File: airbyte-integrations/connectors/source-posthog/metadata.yaml [hunk 1]
```diff
@@ -23,6 +23,13 @@ data:
       enabled: true
   releaseStage: beta
   supportLevel: community
+  releases:
+    breakingChanges:
+      1.0.0:
+        message:
+          The `event` field in the `events` stream has been corrected to the proper data type.
+          It was previously set as `[object, string]`, but the correct type is only `string`.
```
Comment (katmarkham) [inline airbyte-integrations/connectors/source-posthog/metadata.yaml:31]: The `event` field in the `events` stream has been corrected to the proper data type. To apply this change, refresh the schema for the 'events' stream and reset your data. For more information visit <link to migration guide>

**Review by katmarkham — CHANGES_REQUESTED** [review]

**Review by marcosmarxm — APPROVED** [review]
Thanks @dynalz

**Review by katmarkham — APPROVED** [review]

**Pull request merged**